content/docs/bdd/better-gherkin.md

```diff
@@ -35,3 +35,47 @@ In fact, a good question to ask yourself when writing a feature clause is: “Wi
 
 If the answer is “Yes”, then you should rework it avoiding implementation specific details.
 As a side benefit, in consequence your scenarios will be a lot shorter and much easier to follow and understand.
+
+# Consider a more declarative style
+
+One way to make tests easier to maintain and less brittle is to use a declarative style. Declarative style tests state ideas, rather than details of the UI. They are more informative, they can help tests read better as "living documentation". They are less brittle, and let the test have a longer lifespan. A declarative style helps you focus on the value that the customer is getting, rather than the keystrokes they will use,
+
+Imperative tests are communicative, but because they are so closely tied to the mechanics of the current UI, they are often more work to maintain. It's easiest to understand this by looking at examples.
```

Comment:
"more work to maintain." -> Maybe add a sentence here to say "If the implementation changes, the tests need to be updated too" or something similar.

Reply:
Done.